packages/ckan/src/xml.js

```diff
@@ -73,22 +74,22 @@ const toXML = (dataset) => {
                 'dcat:mediaType': serializeTerm(workExample.out(ns.schema.encodingFormat)),
                 'dcat:accessURL': serializeTerm(workExample.out(ns.schema.url)),
                 'dcterms:title': serializeTerm(workExample.out(ns.schema.name)),
-                'dcterms:rights': serializeTerm(copyright),
+                'dcterms:license': serializeTerm(copyright),
                 'dcterms:format': { '#': distributionFormatFromEncoding(workExample.out(ns.schema.encodingFormat)) },
               },
             }))
 
           const publishers = dataset.out(ns.dcterms.publisher)
             .map(publisher => ({
-              'rdf:Description': {
-                'rdfs:label': publisher.value,
+              'foaf:Organization': {
+                'foaf:name': publisher.value,
```

Comment:
In the handbook, the node is an IRI (`rdf:about`). I assume the name would be actual string and not the identifier.

Reply:
Is it something we need to change in the data? Is there an "easy" way to implement something here?

Reply:
The current code somehow suggests that the publisher is an literal.

I think we'd need to handle both options separately and include the labels in the query. That said, I wonder if ODS handles multilingual labels here...

Reply:
True for the litteral, as it's doing `.value` ; but in that case, should we put something in the `rdf:about` attribute?

The handbook is not specifying if it supports multilingual labels in that case 😬

Reply:
I pushed to handle `rdf:about`

Reply:
Great, thanks a lot! 👍